react/components/EmailAndPassword.js

```diff
@@ -141,9 +140,7 @@ class EmailAndPassword extends Component {
                       passwordPlaceholder ||
                       translate('store/login.password.placeholder', intl)
                     }
-                    showPasswordVerificationIntoTooltip={
-                      showPasswordVerificationIntoTooltip
-                    }
+                    disablePasswordVerification="true"
```

Comment:
```suggestion
                    disablePasswordVerification
```

Or was it intended to be a string? Later on, you used in a boolean clause. BTW, `"false"` wouldn't work as expected